Volume/Supporting Views/Header.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Header.swift
+//  Volume
+//
+//  Created by Cameron Russell on 11/15/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+// TODO: Combine w/ publication PR
+struct Header : View {
+    @State var text: String
+    
+    var body : some View {
+        Text(text)
+            .frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
Do you need the `maxWidth: .infinity`?

Reply:
Yeah, or else the width of the `Text` is the minimum size necessary to fit all the words in `text` and will end up being centered on the page.